model-forecasts/component-models/CU_EAKFC_SIRS/EW49-2018-CU_EAKFC_SIRS.csv

```diff
@@ -1,8019 +1,8020 @@
-HHS Region 1,4 wk ahead,Bin,percent,5.4,5.5,0.003341
-HHS Region 1,4 wk ahead,Bin,percent,5.5,5.6,0.008661
-HHS Region 1,4 wk ahead,Bin,percent,5.6,5.7,0.004001
-HHS Region 1,4 wk ahead,Bin,percent,5.7,5.8,0.002
-HHS Region 1,4 wk ahead,Bin,percent,5.8,5.9,0.003321
-HHS Region 1,4 wk ahead,Bin,percent,5.9,6,0.00266
-HHS Region 1,4 wk ahead,Bin,percent,6,6.1,0.003321
-HHS Region 1,4 wk ahead,Bin,percent,6.1,6.2,0.00134
-HHS Region 1,4 wk ahead,Bin,percent,6.2,6.3,0.00066
-HHS Region 1,4 wk ahead,Bin,percent,6.3,6.4,0
-HHS Region 1,4 wk ahead,Bin,percent,6.4,6.5,0.00132
-HHS Region 1,4 wk ahead,Bin,percent,6.5,6.6,0.00132
-HHS Region 1,4 wk ahead,Bin,percent,6.6,6.7,0.00066
-HHS Region 1,4 wk ahead,Bin,percent,6.7,6.8,0.00066
-HHS Region 1,4 wk ahead,Bin,percent,6.8,6.9,0
-HHS Region 1,4 wk ahead,Bin,percent,6.9,7,0.00066
-HHS Region 1,4 wk ahead,Bin,percent,7,7.1,0
-HHS Region 1,4 wk ahead,Bin,percent,7.1,7.2,0
-HHS Region 1,4 wk ahead,Bin,percent,7.2,7.3,0
-HHS Region 1,4 wk ahead,Bin,percent,7.3,7.4,0
-HHS Region 1,4 wk ahead,Bin,percent,7.4,7.5,0
-HHS Region 1,4 wk ahead,Bin,percent,7.5,7.6,0
-HHS Region 1,4 wk ahead,Bin,percent,7.6,7.7,0
-HHS Region 1,4 wk ahead,Bin,percent,7.7,7.8,0
-HHS Region 1,4 wk ahead,Bin,percent,7.8,7.9,0
-HHS Region 1,4 wk ahead,Bin,percent,7.9,8,0
-HHS Region 1,4 wk ahead,Bin,percent,8,8.1,0
-HHS Region 1,4 wk ahead,Bin,percent,8.1,8.2,0
-HHS Region 1,4 wk ahead,Bin,percent,8.2,8.3,0
-HHS Region 1,4 wk ahead,Bin,percent,8.3,8.4,0
-HHS Region 1,4 wk ahead,Bin,percent,8.4,8.5,0
-HHS Region 1,4 wk ahead,Bin,percent,8.5,8.6,0
-HHS Region 1,4 wk ahead,Bin,percent,8.6,8.7,0
-HHS Region 1,4 wk ahead,Bin,percent,8.7,8.8,0
-HHS Region 1,4 wk ahead,Bin,percent,8.8,8.9,0
-HHS Region 1,4 wk ahead,Bin,percent,8.9,9,0.00066
-HHS Region 1,4 wk ahead,Bin,percent,9,9.1,0
-HHS Region 1,4 wk ahead,Bin,percent,9.1,9.2,0
-HHS Region 1,4 wk ahead,Bin,percent,9.2,9.3,0
-HHS Region 1,4 wk ahead,Bin,percent,9.3,9.4,0
-HHS Region 1,4 wk ahead,Bin,percent,9.4,9.5,0
-HHS Region 1,4 wk ahead,Bin,percent,9.5,9.6,0
-HHS Region 1,4 wk ahead,Bin,percent,9.6,9.7,0
-HHS Region 1,4 wk ahead,Bin,percent,9.7,9.8,0
-HHS Region 1,4 wk ahead,Bin,percent,9.8,9.9,0
-HHS Region 1,4 wk ahead,Bin,percent,9.9,10,0
-HHS Region 1,4 wk ahead,Bin,percent,10,10.1,0
-HHS Region 1,4 wk ahead,Bin,percent,10.1,10.2,0
-HHS Region 1,4 wk ahead,Bin,percent,10.2,10.3,0
-HHS Region 1,4 wk ahead,Bin,percent,10.3,10.4,0
-HHS Region 1,4 wk ahead,Bin,percent,10.4,10.5,0
-HHS Region 1,4 wk ahead,Bin,percent,10.5,10.6,0.00066
-HHS Region 1,4 wk ahead,Bin,percent,10.6,10.7,0
-HHS Region 1,4 wk ahead,Bin,percent,10.7,10.8,0
-HHS Region 1,4 wk ahead,Bin,percent,10.8,10.9,0
-HHS Region 1,4 wk ahead,Bin,percent,10.9,11,0
-HHS Region 1,4 wk ahead,Bin,percent,11,11.1,0
-HHS Region 1,4 wk ahead,Bin,percent,11.1,11.2,0
-HHS Region 1,4 wk ahead,Bin,percent,11.2,11.3,0
-HHS Region 1,4 wk ahead,Bin,percent,11.3,11.4,0
-HHS Region 1,4 wk ahead,Bin
```

Comment:
I think that this line is supposed to be "HHS Region 6" right?

Reply:
sorry - fixed now.  i submitted it as a new pull request since this one had already been merged.